package.json

```diff
@@ -2,31 +2,60 @@
   "name": "juggernaut-demo",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://juggernaut-demo.firebaseapp.com/",
   "dependencies": {
-    "@material-ui/core": "^4.3.2",
-    "@material-ui/icons": "^4.2.1",
-    "@material-ui/styles": "^4.3.0",
-    "axios": "^0.18.0",
-    "formik": "^1.5.8",
-    "lodash.defaultsdeep": "^4.6.0",
+    "@material-ui/core": "^4.9.3",
+    "@material-ui/icons": "^4.9.1",
+    "@mdx-js/mdx": "^1.6.18",
+    "axios": "^0.20.0",
+    "i18next": "^19.8.1",
+    "i18next-browser-languagedetector": "^6.0.1",
+    "lodash.defaultsdeep": "^4.6.1",
     "prop-types": "^15.7.2",
-    "qs": "^6.6.0",
-    "react": "^16.8.2",
-    "react-dom": "^16.8.2",
-    "react-router-dom": "^5.0.0",
-    "react-scripts": "3.0.0",
-    "styled-components": "^4.1.3",
-    "yup": "^0.27.0"
+    "qs": "^6.9.1",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-i18next": "^11.7.3",
+    "react-router-dom": "^5.1.2",
+    "react-scripts": "^3.4.3",
```

Comment:
May we update React-Scripts for the latest 4.0.1?

Reply:
You're right, but I think is better to do this in another PR.